### ISSUE TITLE
Remove automatic update upon language switching

### ISSUE THREAD

**Issue by ghost** [yomidevs/yomitan#879]
I would like to use Yomitan itself to scan text inside the Yomitan popup/window. Unfortunately, when you change the language, the popup automatically updates the dictionary info, so this use case doesn't work at the moment.

It would be great if this use case could be supported.

https://github.com/themoeway/yomitan/assets/103891807/924dd37a-aa5b-4423-8b99-1df1be67de01

**Comment by StefanVukovic99** [comment]
![image](https://github.com/themoeway/yomitan/assets/24891609/e7e7ff29-5182-4974-a192-9f53b53dab12)
I think the profile conditions should let you scan your japanese popup with your english profile?

**Comment by RyanOrigens** [comment]
Oh, I wasn't aware of that. It seems like a very interesting feature. Unfortunately, though, it wouldn't work out. If I set the Chrome extension URL to scan English, I wouldn't be capable of scanning other Japanese text present in the example sentence. Additionally, I wouldn't be able to use the search bar with Japanese characters. The pop-up level wouldn't work either if I tried to scan Japanese text.

Example (I would be able to scan one or the other)
![image](https://github.com/themoeway/yomitan/assets/103891807/2824fbd5-11a4-4f76-8c9b-58c3db85c610)

**Comment by StefanVukovic99** [comment]
I was thinking of the popup-level, though if you want to be able to scan the first popup with either profile, maybe you could just set them to different keys?

**Comment by RyanOrigens** [comment]
Yes, you're right. I could utilize another profile key. But let's imagine I'm in a different scenario where I also have a Korean-to-Japanese dictionary containing Korean and Japanese sentences in its example sentence. In that scenario, I would have to set one extra key to be able to scan this dictionary. Let's say my default is 'Super,' and then I would have to utilize another two key like 'Alt' and 'Shift.' You could argue that maybe it could be worked out  with a preference order and stick with only two keys. But my point here is, don't you think other users may also come across this use case, and it would be too hacky of a solution? Perhaps there could be a better way?"

**Comment by StefanVukovic99** [comment]
I see what you mean, the hotkeys don't cover such cases elegantly... Maybe there is a way to cover this without damaging other use cases, but I can't think of any.

**Comment by Kuuuube** [comment]
Closing as this request is as much of a feature for some people as it would be a bug for others.
